libs/dmlf/tests/dmlf_muddle/dmlf_muddle.cpp

```diff
@@ -0,0 +1,216 @@
+//------------------------------------------------------------------------------
```

Comment:
We put the tests in tests/execution maybe this could also got there?

Reply:
There's also an argument that they should go in tests/networking...